pandas/tests/frame/test_apply.py

```diff
@@ -635,3 +635,46 @@ def test_nuiscance_columns(self):
         expected = DataFrame([[6, 6., 'foobarbaz']],
                              index=['sum'], columns=['A', 'B', 'C'])
         assert_frame_equal(result, expected)
+
+    def test_non_callable_aggregates(self):
+
```

Comment:
can you add a comment of ``.size`` here (and how its a property of Series/DataFrame and hence we allow this)